ckanext/data_qld/templates/admin/config.html

```diff
@@ -19,6 +19,19 @@
 
 {{ form.select('ckan.homepage_style', id='field-homepage-style', label=_('Homepage'), options=homepages, selected=data['ckan.homepage_style'], error=error) }}
 
+{% if h.profanity_checking_enabled() %}
+    {{-
+        form.textarea(
+            'ckan.comments.profanity_list',
+            id='field-ckan-comments-profanity-list',
+            label=_('Comments - custom profanity list (one word per line)'),
+            placeholder=_('e.g.\nshit\ndick\n'),
```

Comment:
Can we use slightly less profane examples?

Reply:
@ThrawnCA :( changed